packages/core/src/pubsub/pubsub.ts

```diff
@@ -71,7 +72,7 @@ export class Pubsub extends Observable<PubsubMessage> {
     return this.peerId$
       .pipe(
         mergeMap(async (peerId) => {
-          await this.ipfs.pubsub.publish(this.topic, serialize(message));
+          await this.ipfs.pubsub.publish(this.topic, uint8ArrayFromString(serialize(message)));
```

Comment:
Guess, this to uint8array convertation should be put into `serialise` function @rvagg